packages/core/data-transfer/lib/engine/index.ts

```diff
@@ -172,7 +172,11 @@ class TransferEngine<
   }
 
   #assertSchemasMatching(sourceSchemas: any, destinationSchemas: any) {
-    const strategy = this.options.schemasMatching || 'strict';
+    const strategy = this.options.schemaStrategy || 'strict';
+    if (strategy === 'ignore') {
```

Comment:
I actually think we should use "exact" here since that's our default. But I do think we should be enforcing our options in the constructor. With Typescript, the defaults won't even be reached because we require explicit values, but since we call this with javascript it's possible we fail to send a valid options object so we should enforce it there instead of checking in every call.